roles/dnsmasq/README.md

```diff
@@ -159,24 +159,29 @@ supported in libvirt).
 
 ### New host parameters
 
-* `cifmw_dnsmasq_host_network`: (String) Existing network name.
-* `cifmw_dnsmasq_host_state`: (String) Host status. Must be either `present` or `absent`.
-* `cifmw_dnsmasq_host_mac`: (String) Host MAC address.
-* `cifmw_dnsmasq_host_ips`: (List) Host IP addressees.
-* `cifmw_dnsmasq_host_name`: (String) Host name. Optional.
+* `cifmw_dnsmasq_dhcp_entries`: (List[mapping]) List of DHCP entries.
+
```

Comment:
could there be folks out there using the 'old' vars ... can/should we do something about them?

Reply:
Little to no chance anyone is using this role outside of the reproducer/libvirt_manager.
If really, they'll get an error with the updated  `ansible.builtin.assert` and will have to update their things.

No way we maintain the old interface, it would make things overcomplicated for no actual value imho